File: CRM/src/app/shared/eHospital_DongNai_A_NSTL/NS_DM_TINH.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_DM_TINH } from './NS_DM_TINH.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_DM_TINHService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'MaTinh', 'TenTinh', 'DangSuDung'];

    List: NS_DM_TINH[] | undefined;
    ListFilter: NS_DM_TINH[] | undefined;
    FormData!: NS_DM_TINH;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_DM_TINH";
    }
}

File: Service_eHospital_DongNai_A_NSTL/Implement/NS_DM_TINHService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_DM_TINHService : BaseService<NS_DM_TINH, INS_DM_TINHRepository>
    , INS_DM_TINHService
    {
    private readonly INS_DM_TINHRepository _NS_DM_TINHRepository;
    public NS_DM_TINHService(INS_DM_TINHRepository NS_DM_TINHRepository) : base(NS_DM_TINHRepository)
    {
    _NS_DM_TINHRepository = NS_DM_TINHRepository;
    }
    }
    }

